extensions/amp-recaptcha-input/0.1/amp-recaptcha-service.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Honestly I feel you won't need two files. But that's totally up to you : )

Reply:
I wanted to split into two in case we end up creating an `<amp-recaptcha-onload>` or something like that. That way the logic is already abstracted 😄

Reply:
Sounds good.